### DISCUSSION TITLE
help me to out

### DISCUSSION THREAD

**Discussion by breakingbad45** [pocketbase/pocketbase#3631] [Q&A]
@request.data.publicDate <= @now - 1 minute 

this is not work.i want to do that if the publicdate or timestamp between currenttimestamp or currenttimestamp -1 minute than accept the req

**Comment by ganigeorgiev** [comment]
I don't understand what is the desired behavior but math expressions are not supported in the filters.

You can use a before request hook to manually inspect the submitted data but considering that your recent questions all seems related to the same security thing you are trying to do I'll not pursue the matter further as I think you are going in the wrong direction and will cause more harm than good.

**Comment by breakingbad45** [comment]
{
id:"hasgfh"
name:"user a"
timestamp:"2023-10-28T14:25:25Z" utc curret time stamp
}


`@request.data.timestamp <= @now  `
is this rule work if my timestamp in utc format?

**Reply by ganigeorgiev** [reply]
The date must be in the RFC3339 format including the ms part, aka. `2023-10-28 14:25:25.999Z` (partial comparisons are also allowed aka. `2023-10-28 14:25:25`).

**Reply by breakingbad45** [reply]
Thanks. Set a rule for a specific users type such that if request.authstore.model.role equals ‘admin’, then the rule @request.data.timestamp <= @now is applicable..is it possible to do?
